pkg/network/http/shared_libraries.go

```diff
@@ -5,25 +5,172 @@ package http
 import (
 	"fmt"
 	"os"
-	"path/filepath"
+	fp "path/filepath"
```

Comment:
I think it would be better to alias the `gopsutil` import instead (maybe `psfilepath`?). If I see `filepath` in the code, my instinct is to think it is the stdlib `path/filepath`.

Reply:
Agreed. I'll change that